layout/theme.liquid

```diff
@@ -18,11 +18,9 @@
 
     {% render 'typography' %}
 
-    {{ 'styles.css' | asset_url | stylesheet_tag }}
+    {% render 'entry' with 'theme.entry.js' %}
```

Comment:
@blanklob  Let's add a leading slash when loading the entry point since it's at the theme's root.

```liquid
{% render 'entry' with '/entry.theme.js' %}
```

Reply:
yep I just did thanks!